src/components/ha-faded.ts

```diff
@@ -0,0 +1,88 @@
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, query, state } from "lit/decorators";
+import { classMap } from "lit/directives/class-map";
+
+@customElement("ha-faded")
+class HaFaded extends LitElement {
+  @property({ type: Number }) public fadedHeight = 84;
+
+  @state() faded = true;
+
+  @state() fadeable = true;
+
+  @query(".container") private _container!: HTMLDivElement;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="container ${classMap({ faded: this.fadeable && this.faded })}"
+        style=${this.faded ? `max-height: ${this.fadedHeight}px` : ""}
+      >
+        <slot></slot>
+      </div>
+      ${this.fadeable
+        ? html`
+            <div
+              class="show ${classMap({ faded: this.fadeable && this.faded })}"
+              @click=${this._toggleContainer}
+            >
+              ${this.faded ? "Show more" : "Show less"}
```

Comment:
Should we even have a "show less" option?

Reply:
If not, we shouldn't have any and just have the text expand

Reply:
removed